src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -524,19 +524,21 @@ public override bool CanVerifySignedPackages(SignedPackageVerifierSettings verif
                 // Please note: Linux/MAC case sensitive for env var name.
                 string signVerifyEnvVariable = _environmentVariableReader.GetEnvironmentVariable("DOTNET_NUGET_SIGNATURE_VERIFICATION");
 
-                // Not opt-out option, only opt-in feature.
+                bool canVerify = RuntimeEnvironmentHelper.IsLinux && X509TrustStore.IsEnabled;
+
                 if (!string.IsNullOrEmpty(signVerifyEnvVariable))
                 {
-                    if (signVerifyEnvVariable.Equals(bool.TrueString.ToUpperInvariant(), StringComparison.Ordinal))
+                    if (string.Equals("TRUE", signVerifyEnvVariable, StringComparison.Ordinal))
```

Comment:
The value for this could be typed by a human, should it be case insensitive?

```suggestion
                    if (string.Equals("TRUE", signVerifyEnvVariable, StringComparison.Ordinal))
                    if (string.Equals(bool.TrueString, signVerifyEnvVariable, StringComparison.OrdinalIgnoreCase))
```

Reply:
Sure.

The case sensitivity is the artifact of a [partially abandoned solution](https://github.com/NuGet/Client.Engineering/issues/882#issuecomment-1124364817).

Case insensitive values for true/false (vs. file paths) follows best practices documented in our OneNote as well as other .NET environment variables.

CC @erdembayar